core/src/soundfont/mod.rs

```diff
@@ -55,14 +56,18 @@ pub trait SoundfontBase: Sync + Send + std::fmt::Debug {
     ) -> Vec<Box<dyn VoiceSpawner>>;
 }
 
+/// Type of sample interpolator
 #[derive(Clone, PartialEq, Eq, Copy, Debug)]
 pub enum Interpolator {
+    /// Nearest neighbor
     Nearest,
+
+    /// Linear
```

Comment:
These comments feel a little redundant, imo either rename Nearest to be NearestNeighbor, or add more description to how each one functions

Reply:
Didn't really know what to write there since explaining how each interpolation method works seemed pointless. Should I add links to Wikipedia articles for each method or something?

Reply:
Future methods might need more explanation if we add more, but yeah these are very simple. Ig you can leave it as-is, though having
```
/// Linear
Linear
```
feels a bit redundant

Reply:
I added the Wikipedia links in the new commit I made. Idk what else to explain there lol